internal/servers/worker/session.go

```diff
@@ -331,6 +349,28 @@ func (w *Worker) makeSessionCloseInfo(
 	return result
 }
 
+// makeFakeSessionCloseInfo makes a "fake" makeFakeSessionCloseInfo, intended
+// for use when we can't contact the controller.
+func (w *Worker) makeFakeSessionCloseInfo(
+	closeInfo map[string]string,
+) map[string][]*pbs.CloseConnectionResponseData {
+	if closeInfo == nil {
+		// Should never happen, panic if it does. Results will be
+		// undefined.
+		panic(errMakeSessionCloseInfoNilCloseInfo)
```

Comment:
Don't put panics into server code, please. Return an error that can be logged.

Reply:
(Please also fix this in `makeSessionCloseInfo`)

Reply:
Now fixed!